### PR TITLE
[blog] mf-apply

### PR DESCRIPTION
**Pull request by bennn**
Advertising the Redex summer school and the new `mf-apply` keyword.

@LeifAndersen @wilbowma @BenChung @MiloDavis

### CODE REVIEW

File: blog/_src/posts/2017-02-27-metafunction-apply.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+You can read more:
+
+- in the "Amb" tutorial: [http://docs.racket-lang.org/redex/tutorial.html](http://docs.racket-lang.org/redex/tutorial.html)
+- in the "Long Tutorial": [http://docs.racket-lang.org/redex/redex2015.html](http://docs.racket-lang.org/redex/redex2015.html)
+- in the Redex reference manual: [http://docs.racket-lang.org/redex/The_Redex_Reference.html](http://docs.racket-lang.org/redex/The_Redex_Reference.html)
+- on the PLT Redex website: [https://redex.racket-lang.org/](https://redex.racket-lang.org/)
+- on GitHub: [https://github.com/racket/redex](https://github.com/racket/redex)
+
+And if you act now, you can become a _Redexionsita_ between July 10 and July 14
```
Comment (gasche) [inline blog/_src/posts/2017-02-27-metafunction-apply.md]: Redexionista?

File: blog/_src/posts/2017-02-27-metafunction-apply.md [hunk 2]
```diff
@@ -0,0 +1,214 @@
+    Title: PLT Redex: mf-apply
+    Date: 2017-02-27T23:54:20
+    Tags: PLT Redex, package, lang-extension, by Ben Greenman
+
+The `mf-apply` keyword is for checked metafunction application in PLT Redex.
+In other words, `(mf-apply f x)` is just like `(f x)`, but errors if `f` is
+ not a previously-defined metafunction.
+
+Also, consider attending _The Racket School of Semantics and Languages_ this summer:
+[http://summer-school.racket-lang.org/2017/](http://summer-school.racket-lang.org/2017/)
```
Comment (gasche) [inline blog/_src/posts/2017-02-27-metafunction-apply.md]: You might want to write an extra sentence that would let someone get a feeling of whether that is for them: what kind of material is covered, in what form, what is the intended public.

File: blog/_src/posts/2017-02-27-metafunction-apply.md (hunk 1 above)
Comment (bennn) [inline blog/_src/posts/2017-02-27-metafunction-apply.md]: Talked with Max, Leif, and Ming-Ho about this as lunch. Some more ideas:
- redexan
- redexhibistionist (my favorite)
- redexist

Comment (gasche) [inline blog/_src/posts/2017-02-27-metafunction-apply.md]: I am fine with redexionista, I just wanted to point out a typo in the text:

```
redexionsita
redexionista
```

Comment (mhyee) [inline blog/_src/posts/2017-02-27-metafunction-apply.md]: When written out, I actually prefer redexan. Short, easy to say, easy to spell.

**Comment by bennn** [comment]
merged! https://github.com/nuprl/website/commit/3292b580ba892a7955cd296b5959cef443ac0efd

**Pull request closed without merging**